packages/ui/src/components/ProjectCard/ProjectCard.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Box, SimpleGrid } from '@chakra-ui/react';
+import { Meta, Story } from '@storybook/react';
+
+import { ProjectCard, ProjectCardProps } from './ProjectCard';
+
+const title = 'Zaawansowany React i Node';
+const date = new Date();
+
+const status = { done: 'done', idle: 'idle', doing: 'doing', review: 'review' };
+const meta: Meta = {
+  title: 'Components / ProjectCard',
+  component: ProjectCard,
+  argTypes: {
```

Comment:
🤔 You should add the `date` prop here and type it correctly, as now Storybook displays it as a string in the controls panel.